### PR TITLE
Boot the stripe integration

### PR DESCRIPTION
**Pull request by claudiaalves**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+You need to install [lein](http://leiningen.org/)
+
+* `lein server` - starts the HTTP server
+
+#### Tests
+
+* `lein test` - runs the test suite
+* `script/autotest` or `lein autotest` -listen for file changes and is always running tests
+* To Run the integration test you should provide a valid Stripe API KEY with access to mb references api.
+  `STRIPE_API_KEY=<YOUR_MEO_WALLET_API_KEY> lein test`
```
Comment (lamuria) [inline README.md]: <MEO_WALLET> ? 😛

File: src/weareswat/stripe_integration/core.clj [hunk 2]
```diff
@@ -0,0 +1,21 @@
+(ns weareswat.stripe-integration.core
+  "This namespace has the main fn that starts the server."
+  (:gen-class)
+  (:require
+    [com.stuartsierra.component :as component]
```
Comment (lamuria) [inline src/weareswat/stripe_integration/core.clj]: you could put this in the same line as the `:require` keyword

File: src/weareswat/stripe_integration/routes.clj [hunk 3]
```diff
@@ -0,0 +1,12 @@
+(ns weareswat.stripe-integration.routes
+  "Specifices the routes"
+  (:require
+    [compojure.core :as compojure :refer [GET]]
```
Comment (lamuria) [inline src/weareswat/stripe_integration/routes.clj]: same here

File: README.md (hunk 1 above)
Comment (claudiaalves) [reply README.md]: Copy paste my man :)

File: src/weareswat/stripe_integration/core.clj (hunk 2 above)
Comment (donbonifacio) [reply src/weareswat/stripe_integration/core.clj]: Actually this is the "standard style" advised on clojure.

**Review by donbonifacio — APPROVED** [review]

**Review by paulosilva86 — APPROVED** [review]

**Review by jmmalaca — APPROVED** [review]

**Pull request merged**